kyuubi-server/web-ui/src/locales/en_US/index.ts

```diff
@@ -37,21 +37,25 @@ export default {
   engine_ui: 'Engine UI',
   failure_reason: 'Failure Reason',
   session_properties: 'Session Properties',
+  no_data: 'No data yet',
```

Comment:
Please correct me if this is incorrect, we return no_data when query success but without empty data, so this should be 'No data'?